javascript/index.js

```diff
@@ -13,53 +13,69 @@ let milDec = document.getElementById('milDec');
 let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
+
 function printTime() {
-  // ... your code goes here
+  printMinutes();
+  printSeconds();
+  printMilliseconds();
 }
 
 function printMinutes() {
-  // ... your code goes here
+  let minsFirstDig = chronometer.twoDigitsNumber(chronometer.getMinutes())[0];
+  let minsSecDig = chronometer.twoDigitsNumber(chronometer.getMinutes())[1];
+  document.querySelector('#minDec').innerText = minsFirstDig;
```

Comment:
In lines 4 to 14 you already have the reference of all the DOM elements you will be modifying, so you can target those variables instead of selecting the DOM elements again.